src/androidfde_fmt_plug.c

```diff
@@ -62,7 +62,7 @@ john_register_one(&fmt_fde);
 #define BENCHMARK_LENGTH    0x107
 #define BINARY_SIZE         0
 #define BINARY_ALIGN        1
-#define SALT_ALIGN          8
+#define SALT_ALIGN          sizeof(struct custom_salt*)
 #define SALT_SIZE           sizeof(struct custom_salt)
```

Comment:
I think it's confusing to use `sizeof(struct custom_salt*)` here because that's not the pointer type found _in_ the struct. We actually need alignment because of the `unsigned char *cipherbuf;` field here (BTW, the struct could be made smaller by re-ordering fields, not keeping this pointer between `int` fields). So I think it's better to use `sizeof(void *)` or `sizeof(unsigned char *)`.